packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -614,8 +614,8 @@ abstract class FlutterCommand extends Command<void> {
       FlutterOptions.kDartDefinesOption,
       aliases: <String>[ kDartDefines ], // supported for historical reasons
       help: 'Additional key-value pairs that will be available as constants '
-            'from the String.fromEnvironment, bool.fromEnvironment, int.fromEnvironment, '
-            'and double.fromEnvironment constructors.\n'
+            'from the String.fromEnvironment, bool.fromEnvironment and int.fromEnvironment '
+            'constructors.\n'
```

Comment:
Nit: Consider preserving the serial/oxford comma to stay consistent with the rest of this file. 

```suggestion
            'from the String.fromEnvironment, bool.fromEnvironment, and int.fromEnvironment '
            'constructors.\n'
```